<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Trans</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="inputEditor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputEditor">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>381</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="transButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>99</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>翻译</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>540</y>
      <width>99</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
